subprojects/gradle/runner/service/src/main/kotlin/com/avito/runner/service/worker/device/adb/listener/AdbDeviceMetrics.kt

```diff
@@ -0,0 +1,126 @@
+package com.avito.runner.service.worker.device.adb.listener
+
+import com.avito.android.stats.CountMetric
+import com.avito.android.stats.StatsDSender
+import java.nio.file.Path
+
+class AdbDeviceMetrics(
+    private val statsDSender: StatsDSender,
+    buildId: String,
+    instrumentationConfigName: String
+) : AdbDeviceEventsListener {
+
+    private val prefix = "testrunner.$buildId.$instrumentationConfigName.adb"
```

Comment:
This concatenation details look like leaky abstraction.

- Is `testrunner` a commom prefix for all metrics from test runner? If so, we can add one time in StatsD client instance.
- `<buildId>.<configuration name>` seems kind of "Build Configuration Prefix / Identifier". Propagating `configurationName` through different layers of abstration is a marker for it.

Maybe it's worth to isolate such details if `AdbDeviceMetrics` is only a one of multiple metrics producers in runner.
up to you